src/main/java/view/InputView.java

```diff
@@ -0,0 +1,34 @@
+package view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+    private static final Scanner scanner = new Scanner(System.in);
+    private static final String NAME_INPUT_REQUEST = "참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)";
+    private static final String DELIMITER = ",";
+    private static final String HEIGHT_INPUT_REQUEST = "최대 사다리 높이는 몇 개인가요?";
+
+    private final InputPlayerNameValidator inputPlayerNameValidator = new InputPlayerNameValidator();
+    private final InputHeightValidator inputHeightValidator = new InputHeightValidator();
+
+    public List<String> readUserNames() {
+        System.out.println(NAME_INPUT_REQUEST);
+        String inputUserNames = scanner.nextLine();
+        List<String> playersNames = splitInputUserNames(inputUserNames);
+        inputPlayerNameValidator.checkPlayer(playersNames);
+        return playersNames;
+    }
+
+    public int readHeight() {
+        System.out.println(HEIGHT_INPUT_REQUEST);
+        String inputHeight = scanner.nextLine();
+        inputHeightValidator.checkHeight(inputHeight);
+        return Integer.parseInt(inputHeight);
```

Comment:
parseInt 이전에 parseInt의 validation들을 진행하고 있네요! (null check, empty check, regex 체크) 아래와 같이 진행해보는 것은 어떨까요?

다른 부분도 validation들을 조금 더 간결하게 진행할 수 있을 것 같아요. (validation 비용도 최적화한다는 관점으로 고민해 보시면 좋을 것 같아요)

```suggestion
        String inputHeight = scanner.nextLine();
        Integer height = try {
            Integer.parseInt(inputHeight);
        } throw new IllegalArgumentException()
        return height
```